spec/decorators/user_decorator_spec.rb

```diff
@@ -0,0 +1,101 @@
+require "rails_helper"
```

Comment:
this is new because all these decorator tests where inside `user_spec.rb` and I moved them out